### PR TITLE
Feature/auth

### PR DESCRIPTION
**Pull request by GemPearl**
Updated the User.Js with registration auth. and linking the registration to Postgress.

### CODE REVIEW

File: server/routes/auth.js [hunk 1]
```diff
@@ -12,10 +12,13 @@ router.post("/login",passport.authenticate('local'), (req, res)=>{
     res.json(req.session.passport.user)
 })
 
+<<<<<<< HEAD
+=======
 router.post("/logout", (req, res)=>{
     req.logout();
     res.status(200).send({msg:"log out successfully"})
 })
+>>>>>>> 920bffcafe3212aaf55631370f5389931a4d6526
```
Comment (LiamGve) [inline server/routes/auth.js:21]: This looks like you are committing a merge conflict. Please remove this line and also lines `15` and `16`

**Review by LiamGve — COMMENTED** [review]
This PR looks good but before we merge I think we need to double check the indentation and stylings in the files that have been worked on

File: package.json [hunk 2]
```diff
@@ -1,97 +1,88 @@
...
+    "rimraf": "^3.0.2",
+    "style-loader": "^3.3.1",
+    "wait-on": "^6.0.0",
+    "webpack": "^5.68.0",
+    "webpack-cli": "^4.9.2",
+    "webpack-dev-server": "^4.7.4",
+    "webpack-merge": "^5.8.0"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.17.0",
```
Comment (LiamGve) [inline package.json:71]: I think something has gone wrong here as we see to have removed a lot of the dependencies. Was this intentional?

Comment (GemPearl) [reply package.json:71]: I am looking into it now. I accepted the new changes, did not intend to remove any dependencies.

**Review by LiamGve — APPROVED** [review]

**Pull request merged**